### ISSUE TITLE
Files 'own data' is leaking to the next compiled file

### ISSUE THREAD

**Issue by erikljung** [ahmadnassri/node-metalsmith-pug#4]
First I like to say thank you for a nice work! Second I would like to say that there is a possibility that i might have misunderstood the concept of what I considered a bug. Thanks for your help!
## Problem

If you have two files (one.jade, two.jade) looking like following
**_one.jade**_

```

---
title: file one
someValue: true

---
p file one
```

**_two.jade**_

```

---
title: file two

---
p file two
p= someValue
```

The **_someValue**_ will be true in **_two.jade**_.
My expectation was that file data should be isolated to the current rendering file.
## Solution

**Line 45 in index.js** 

```
// also use the file's own data
if (opts.useMetadata) {
    fileLocals = util._extend(util._extend({}, opts.locals), data);   
}
```

**Line 58 in index.js**

```
// render
var str = jade.compile(data.contents.toString(), opts)(fileLocals)
```

**Comment by ahmadnassri** [comment]
that is indeed a bug! I'll pull the fix from your repo.

**Comment by ahmadnassri** [comment]
released in `v2.0.1`

**Comment by erikljung** [comment]
Nice, thank you!
